packages/frontend/.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: '../../.eslintrc.json',
+  extends: ['../../.eslintrc.json', 'plugin:storybook/recommended'],
```

Comment:
why is this needed?

Reply:
to make use of the storybook eslint plugin